Apps/phmimecast/mimecast_connector.py

```diff
@@ -353,33 +351,43 @@ def _make_rest_call(self, endpoint, action_result, headers=None, params=None, da
                             json=data,
                             verify=config.get('verify_server_cert', False),
                             params=params)
+        except requests.exceptions.InvalidSchema:
```

Comment:
Is there a reason why string formatting changed to the older version here? You used .format everywhere else.

Reply:
Nope. Fixed it